robot_controllers/src/parallel_gripper.cpp

```diff
@@ -59,9 +59,27 @@ int ParallelGripperController::init(ros::NodeHandle& nh, ControllerManager* mana
   std::string l_name, r_name;
   nh.param<std::string>("l_gripper_joint", l_name, "l_gripper_finger_joint");
   nh.param<std::string>("r_gripper_joint", r_name, "r_gripper_finger_joint");
+
   left_ = manager_->getJointHandle(l_name);
   right_ = manager_->getJointHandle(r_name);
 
+  // Checking to see if Joint Handles exists
+  if (!left_)
+  {
+    ROS_ERROR_NAMED("ParallelGripperController",
+                     "Unable to retreive joint (%s), Namespace: %s/l_gripper_joint",
+                                          l_name.c_str(),nh.getNamespace().c_str());
```

Comment:
missing spaces after commas. Also, all three lines should probably be vertically aligned (that is, "Parallel.. is aligned with "Unable... is aligned with l_name.c_str()...). Same issue below. Please fix this up this morning so we can merge and release (also note derek's comment above)